pkg/azurefile/nodeserver.go

```diff
@@ -56,6 +61,10 @@ func (d *Driver) NodePublishVolume(ctx context.Context, req *csi.NodePublishVolu
 		return nil, status.Error(codes.InvalidArgument, "Staging target not provided")
 	}
 
+	volumeID := req.GetVolumeId()
+	volumePublishInProgress[volumeID] = true
+	defer delete(volumePublishInProgress, volumeID)
```

Comment:
what about there are two `NodePublishVolume` calls in same time, shall we handle that?

Reply:
could refer to https://github.com/kubernetes-sigs/gcp-compute-persistent-disk-csi-driver/blob/2014365ea7b3405e024be1d9552aa30d44da4a8b/pkg/gce-pd-csi-driver/node.go#L108-L110

Reply:
sure.